api/src/main/java/org/ccci/idm/user/DefaultUserManager.java

```diff
@@ -34,12 +36,20 @@
     private static final EmailValidator VALIDATOR_EMAIL = EmailValidator.getInstance();
 
     @NotNull
+    @Autowired(required = false)
```

Comment:
it would be possible to shift the config to explicit XML and drop the `@Autowired` annotation, the benefit of using Autowired with a collection is that it will automatically pick up all Listeners without having to define the list of them in the spring XML.

So, with the current Key code I have a Listener for syncing changes to the GR, and I will be implementing one to sync password changes as well. Without `@Autowired` I would manually have to list the listeners I want to use, with `@Autowired` I just have to create the bean in the GR config and the USDS policies config and they will automatically be used.

`required = false` makes it so that the DefaultUserManager will continue to work without having any defined listeners.